### PR TITLE
translation beginner_source/translation_transformer

### PR DESCRIPTION
**Pull request by dudtheheaven**
## 라이선스 동의
_변경해주시는 내용에 BSD 3항 라이선스가 적용됨을 동의해주셔야 합니다._<br />
_더 자세한 내용은 [기여하기 문서](https://github.com/PyTorchKorea/tutorials-kr/blob/master/CONTRIBUTING.md)를 참고해주세요._<br />
_동의하시면 아래 `[ ]`를 `[x]`로 만들어주세요._<br />

- [x] 기여하기 문서를 확인하였으며, 본 PR 내용에 BSD 3항 라이선스가 적용됨에 동의합니다.

## 관련 이슈 번호
_이 Pull Request와 관련있는 이슈 번호를 적어주세요._<br />
_이슈 또는 PR 번호 앞에 #을 붙이시면 제목을 바로 확인하실 수 있습니다. (예. #999 )_

- **이슈 번호**: #704

## PR 종류
_이 PR에 해당되는 종류 앞의 `[ ]`을 `[x]`로 변경해주세요._<br />
- [x] 오탈자를 수정하거나 번역을 개선하는 기여
- [ ] 번역되지 않은 튜토리얼을 번역하는 기여
- [ ] 공식 튜토리얼 내용을 반영하는 기여
- [ ] 위 종류에 포함되지 않는 기여

## PR 설명
_이 PR로 무엇이 달라지는지 대략적으로 알려주세요._

번역 개선

마지막으로 Transformer 모델의 출력을 선형 계층에 통과시켜 도착어의 각 토큰에 대한 정규화되지 않은 확률(un-normalized probability)로 제공합니다.

→ 마지막으로 Transformer 모델의 출력을 선형 계층에 통과시켜 도착어의 각 토큰에 ‘대해’ 정규화되지 않은 확률(un-normalized probability)로 제공합니다.

### CODE REVIEW

File: beginner_source/translation_transformer.py [hunk 1]
```diff
@@ -96,7 +96,7 @@ def yield_tokens(data_iter: Iterable, language: str) -> List[str]:
 # 이러한 임베딩은 입력 토큰의 위치 정보(position information)를 모델에 전달하기 위해
 # 위치 인코딩(positional encoding)을 추가합니다.
 # 두번째 부분은 실제 `Transformer <https://pytorch.org/docs/stable/generated/torch.nn.Transformer.html>`__ 모델입니다.
```
Comment (chanmuzi) [inline beginner_source/translation_transformer.py:98]: '두번째'도 '두 번째'로 고치면 좋을 것 같습니다!

File: beginner_source/translation_transformer.py [hunk 2]
```diff
@@ -96,7 +96,7 @@ def yield_tokens(data_iter: Iterable, language: str) -> List[str]:
...
-# 마지막으로 Transformer 모델의 출력을 선형 계층에 통과시켜 도착어의 각 토큰에 대한 정규화되지 않은
+# 마지막으로 Transformer 모델의 출력을 선형 계층에 통과시켜 도착어의 각 토큰에 대해 정규화되지 않은
```
Comment (chanmuzi) [inline beginner_source/translation_transformer.py:99]: 지금처럼 '각 토큰에 대해'라고 하는 것도 좋고, '각 토큰의'로도 수정 가능할 것 같아요!

**Review by chanmuzi — APPROVED** [review]
수정 잘 하신 것 같습니다!

File: beginner_source/translation_transformer.py (hunk 1 above)
Comment (dudtheheaven) [reply beginner_source/translation_transformer.py:98]: 네 반영해서 다시 수정하겠습니다!

File: beginner_source/translation_transformer.py (hunk 2 above)
Comment (dudtheheaven) [reply beginner_source/translation_transformer.py:99]: 넵 참고해보겠습니다!

**Review by hyoyoung — APPROVED** [review]
good

**Review by hyoyoung — CHANGES_REQUESTED** [review]
추가된 submodule 확인 바랍니다

**Comment by khleexv** [comment]
`git rm` 같은 명령으로 submodule 제거할 수 있을 것 같은데, 한번 확인해보시면 좋을 것 같아요 !!

**Comment by dudtheheaven** [comment]
추가된 submodule 삭제했습니다.
